resources/process_workflow.md

```diff
@@ -35,6 +35,28 @@ Be sure not to create a task such as "create a input field", or "create an API",
 
 All SMART tasks should be linked to a User Story, which should be properly detailed with its goal (it can be written using the User Story pattern), Acceptance criteria, and BDD if necessary.
 
+We are following Gherkin syntax to describe our User Stories:
+
+### Title:
+```
+[Verb] [What] [Where]
+
+Examples:
+- Fix typo in Dangerzone
+- Investigate problems in GitLab Sync
+- Add search field to repositories listing in Dangerzone
+```
+
+### Description:
+```
+- Scenario:
+- Given
+- And
+- When
+- Then
+
```

Comment:
**suggestion:** removing this additional empty line

Reply:
Done in ba05e0e